include/deal.II/fe/fe_series.h

```diff
@@ -96,24 +96,29 @@ namespace FESeries
      * each direction, @p fe_collection is the hp::FECollection for which
      * expansion will be used and @p q_collection is the hp::QCollection used to
      * integrate the expansion for each FiniteElement in @p fe_collection.
+     * @p component defines which component of each FiniteElement will be used,
+     * it is mainly used for non-primitive elements.
```

Comment:
Here and other occasions:
```suggestion
     *
     * As the Fourier expansion can only be performed on scalar fields, this class does not operate on vector-valued finite elements and will therefore throw an assertion. However, each component of a finite element field can be treated as a scalar field, respectively, on which Fourier expansions are again possible. For this purpose, the optional parameter @p component defines which component of each FiniteElement will be used.
```